File: python/rosettacode_org/General_FizzBuzz/General_FizzBuzz.py

def genfizzbuzz(factorwords, numbers):
    factorwords.sort(key=lambda p: p[0])
    lines = []
    for num in numbers:
        words = ''.join(wrd for fact, wrd in factorwords if ( num % fact ) == 0)
        lines.append(words if words else str(num))
    return '\n'.join(lines)

if __name__ == '__main__':
    print(genfizzbuzz([(5, 'Buzz'), (3, 'Fizz'), (4, 'Baxx')], range(1, 21)))

File: python/rosettacode_org/General_FizzBuzz/General_FizzBuzz.js

function fizz(d, e) {
  return function b(a) {
    return a ? b(a -1).concat(a) : [];
  }(e).reduce(function(b, a){
    return b + (d.reduce(function(b, c){
      return b + (a % c[0] ? "" : c[1]);
    }, "") || a.toString()) + "\n";
  }, "");
}

console.log(fizz([[3, 'Fizz'], [5, 'Buzz'], [7, 'Baxx']], 20));

